### ISSUE TITLE
[Accessibility] Posts block - Featured Image missing alt text

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#3256]
**Describe the bug**
- When evaluating the page with Posts block in the WAVE Tool, this error comes up:

<img width="1674" alt="Screenshot 2024-07-19 at 1 33 35 PM" src="https://github.com/user-attachments/assets/583b91b6-7792-4443-b9d9-1d66e91f58fe">

- Problem: Posts block > Featured Images is missing alt text
- Sample WAVE tool report: https://wave.webaim.org/report#/https://gambitsandbox.instawp.xyz/andrea/posts-block-accessibility/
- Sandbox page: https://gambitsandbox.instawp.xyz/andrea/posts-block-accessibility/

**To Reproduce**
1. Add a new page
2. Add a Posts block
3. Save page and view frontend
4. Inspect any featured image from the Posts block
5. See if alt text is missing

**Expected behavior**
- Add alt attribute for Posts block > Featured Image
- Alt text should be the Post Title (e.g. `alt="post test"`)
- If no Post Title, it should only show `alt`

**Screenshots**

https://github.com/user-attachments/assets/7da3a4ff-fa76-43e3-8be1-3f532850f1c6
